### DISCUSSION TITLE
Feature Request: A <Table/> component for generating tables conveniently

### DISCUSSION THREAD

**Discussion by ZaeemKhaliq** [diegomura/react-pdf#2144] [Ideas]
Since tables are the core of many PDF reports, a built-in `<Table/>` like component that just takes data (headings, rows etc.) as input and generates the table would be a nice convenient wrapper to avoid building tables from scratch. 

I would be happy to contribute for this if I know how to.

**Comment by staatzstreich** [comment]
Maybee you find that useful...

- https://kags.me.ke/post/generate-dynamic-pdf-incoice-using-react-pdf/
- https://github.com/dmk99/react-pdf-table

**Reply by jeetiss** [reply]
I have this one:
 - [table with react-context ](https://react-pdf-repl.vercel.app/?cp_code=JYWwDg9gTgLgBAbzgZRgTwDYFNkAstYwA0cAIhAMYCuIWAdsXAAoCGA5liQGrBYDuJACpYAHvAC-cAGZQIIOACIAAlCwsKMALRgAJlID0qujqyqoCgNwAoUJFiI4FVSxhYAwhAajGVAM7vPVzESN1xgDB0jEgoMTywAUWxaBjhJGTlFZw1LKysKT194QswsXzgAXhR0bDwCGAA6JzVXAAoEKzg4ACMIHTQALkQOzrgwFh0dYDo2QYBGAAYiYfEl8QBKazyC-EEWLuwPLzEKx2dXQ6CYNvXN_LpCuF39rBO2xzCIqLhfYAAvUtSawqAD44C1hgAeJ4HQLeepMWQAN2AJigcERLAwVCw5QQSB-_zK4nEwOGnQhPH432qOLx3WgqIAQhAYDA5HMSD0oKiAErANi4GBzVIkhAUD6RejiCH6Sl8Unk_TQgJHBoIiDI1Gkja5O4PHkQPivJDi8KSuiAkFDTp6-D4v4AySVPwqy4tZUXbw6zrAKRggCEBNKQJguFkRviUFkUBaCnGOm-DrKYFkYDgbMee2wjjkkDo9BgCm9cFUMCoUAt4JGcApvCNxWwuKQUmwIlIwFUGmAnkGCnDChFCuriFCZqM9RALDALVNn3oJBnEpIUxMIiB5SHw8csXziSwySus50JHt_0GQd8AG0V6IALqAskjNbiR8yuVDnXiW7bOBuLAYDBjXeMd50Tf5LQ3MFITlR8G1pdotzGCYphmRQAFY4AWBQli3FtREGAADeY4GIgASBAg3EABSAicOHLlUUECAwA5R9OgY0wABksCkIVMLozpiWGTca2EMRgTFCUjGlJVvCHN8621TZRDseATCkFgqAweAWnXUEqxrchqAPESIVYDgwJxBQAEEABYB2gXgGBcbs6HKBQMBYYxfAoKcsAHODcTg3x6h6PoSTYms5WpEomzgTF-ToABJVwQF8Xs8JETRChYWAB2JETqyhLMXgvXFLwANhIKq4AAJgADhIABmRY4Ba28Iq3IqDXlSKtwhP8AOBWYZUGjBCq6gb_3G2rRumib-rG4Emrmoa-uHKahts1bxvWoqlrQnaFpGGUeuO8kzr2k6lpG_Qlqu8kltmu75oemslpWl61q6jalu2r7dp-_b5sOgHzprfRLqBmsoeh975tu-64fhobnqRuHNvGgAJaaIDgPhoAiaRZHkFh0xKo63sx4F_vR6HqdBunJshw1wYhWH6ZuynkeptHXp5j7uYxpacYAvGCagIn0lJ8nniFzmQfl5mOcmlWfupxH-eF-a-e-7Whs-pn1b-pXjfm0XYnxwmE2l-LZewU2NpZ3r1aVEqJoU_hTP0cysHk_QjJoAslKsIA&modules=true)
